src/dwalk/dwalk.c

```diff
@@ -498,9 +498,6 @@ int main(int argc, char** argv)
         mfu_flist_print(flist);
     }
 
-    /* print summary about all files */
-    mfu_flist_print_summary(flist);
```

Comment:
Actually, this line is still needed to print the summary.  The mfu_flist_print() call above prints info about individual files, whereas mfu_flist_print_summary() prints summary information about the list as a whole.